Root_Loader.php

```diff
@@ -20,6 +20,7 @@ function __construct() {
 
 		$plugins = array();
 		$plugins[] = new Generic_Plugin();
+		$plugins[] = new UserExperience_Plugin();
```

Comment:
this should be conditional to not load files (and waste compilation+execution time and memory) when they are not used.

see e.g. https://github.com/W3EDGE/w3-total-cache/blob/583d7a056f0d19345ac8045ec175c42b4ba414d1/Root_Loader.php#L35

Reply:
I renamed the new class and conditionally loaded it.